test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,61 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+    it('Consume GET Service', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+
+        const response = await axios.get('https://httpbin.org/get', { query });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+    it('Consume HEAD Service', async () => {
+        const response = await axios.head('https://httpbin.org/headers');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.headers).to.have.property("content-type");
+        expect(response.data).to.eql("")
+
+    })
+
+    it('Consume PATCH Service', async () => {
+        const response = await axios.patch("https://httpbin.org/patch", query);
```

Comment:
2 problems:
1. The variable query is being defined inside another test (it). On those cases, you should not re use it on other tests because they should be independent. So, if you run only this it, it will fail because the variable query would not be defined.

2. For PATCH requests, the parameter received is not a `query`, but a `body`, so consider naming the variable according to what it really represents for this case

Reply:
Hi , i fixed.